### PR TITLE
cleaned up integration-cli/docker_api_containers_test.go

### PR DESCRIPTION
**Pull request by ZJaffee**
As part of "Proposal: use of checkers on Integration test #16756"

### CODE REVIEW

File: integration-cli/docker_api_containers_test.go [hunk 1]
```diff
@@ -1268,20 +1141,18 @@ func (s *DockerSuite) TestContainerApiDeleteRemoveVolume(c *check.C) {
...
 	status, _, err := sockRequest("DELETE", "/containers/"+id+"?v=1&force=1", nil)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusNoContent)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusNoContent)
 
-	if _, err := os.Stat(source); !os.IsNotExist(err) {
-		c.Fatalf("expected to get ErrNotExist error, got %v", err)
-	}
-}
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Feels like you delete one `}` too much.

File: integration-cli/docker_api_containers_test.go [hunk 2]
```diff
@@ -24,31 +25,24 @@ import (
 func (s *DockerSuite) TestContainerApiGetAll(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	startCount, err := getContainerCount()
-	if err != nil {
-		c.Fatalf("Cannot query container count: %v", err)
-	}
+	c.Assert(err,checker.IsNil,Commentf("Cannot query container count: %v", err))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Here and everywhere else in this file: s/Commentf/check.Commentf/

Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Also, here : `c.Assert(err, checker.IsNil, check.Commentf("Cannot query container count"))`

File: integration-cli/docker_api_containers_test.go [hunk 3]
```diff
@@ -24,31 +25,24 @@ import (
...
 		Names []string
 	}
-	if err = json.Unmarshal(body, &inspectJSON); err != nil {
-		c.Fatalf("unable to unmarshal response body: %v", err)
-	}
 
-	if len(inspectJSON) != startCount+1 {
-		c.Fatalf("Expected %d container(s), %d found (started with: %d)", startCount+1, len(inspectJSON), startCount)
-	}
+	c.Assert(json.Unmarshal(body, &inspectJSON),checker.IsNil,Commentf("unable to unmarshal response body: %v", json.Unmarshal(body, &inspectJSON)))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: This ain't gonna work, you call twice `json.Unmarshal` but I don't even think you need a comment here.

File: integration-cli/docker_api_containers_test.go [hunk 4]
```diff
@@ -24,31 +25,24 @@ import (
...
 
-	if actual := inspectJSON[0].Names[0]; actual != "/"+name {
-		c.Fatalf("Container Name mismatch. Expected: %q, received: %q\n", "/"+name, actual)
-	}
+	c.Assert(len(inspectJSON),checker.Equals,startCount+1,Commentf("Expected %d container(s), %d found (started with: %d)", startCount+1, len(inspectJSON), startCount))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: `checker.HasLen` -> `c.Assert(inspectJSON,checker.HasLen,startCount+1)`

File: integration-cli/docker_api_containers_test.go [hunk 5]
```diff
@@ -24,31 +25,24 @@ import (
...
+
+	c.Assert(inspectJSON[0].Names[0], checker.Equals,"/"+name,Commentf("Container Name mismatch. Expected: %q, received: %q\n", "/"+name, inspectJSON[0].Names[0]))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: `c.Assert(inspectJSON[0].Names[0], checker.Equals,"/"+name)`

File: integration-cli/docker_api_containers_test.go [hunk 6]
```diff
@@ -77,9 +71,7 @@ func (s *DockerSuite) TestContainerApiGetJSONNoFieldsOmitted(c *check.C) {
 	// decoding into types.Container do not work since it eventually unmarshal
 	// and empty field to an empty go map, so we just check for a string
 	for _, f := range fields {
-		if !strings.Contains(string(body), f) {
-			c.Fatalf("Field %s is missing and it shouldn't", f)
-		}
+		c.Assert(!strings.Contains(string(body), f),checker.Equals,true,Commentf("Field %s is missing and it shouldn't", f))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: `c.Assert(string(body), checker.Contains, f)`

File: integration-cli/docker_api_containers_test.go [hunk 7]
```diff
@@ -156,16 +146,15 @@ func (s *DockerSuite) TestContainerApiGetChanges(c *check.C) {
...
 
 	changes := []struct {
 		Kind int
 		Path string
 	}{}
-	if err = json.Unmarshal(body, &changes); err != nil {
-		c.Fatalf("unable to unmarshal response body: %v", err)
-	}
+
+	c.Assert(json.Unmarshal(body, &changes),checker.IsNil,Commentf("unable to unmarshal response body: %v",json.Unmarshal(body, &changes)))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Same as above

File: integration-cli/docker_api_containers_test.go [hunk 8]
```diff
@@ -229,12 +213,10 @@ func (s *DockerSuite) TestContainerApiStartDupVolumeBinds(c *check.C) {
...
 	status, body, err := sockRequest("POST", "/containers/"+name+"/start", config)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusInternalServerError)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusInternalServerError)
 
-	if !strings.Contains(string(body), "Duplicate bind") {
-		c.Fatalf("Expected failure due to duplicate bind mounts to same path, instead got: %q with error: %v", string(body), err)
-	}
+	c.Assert(!strings.Contains(string(body), "Duplicate bind"),checker.Equals,true,Commentf("Expected failure due to duplicate bind mounts to same path, instead got: %q with error: %v", string(body), err))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: `c.Assert(string(body), checker.Contains, "Duplicate bind", …)`

File: integration-cli/docker_api_containers_test.go [hunk 9]
```diff
@@ -483,39 +449,33 @@ func (s *DockerSuite) TestContainerApiPause(c *check.C) {
...
-	c.Assert(status, check.Equals, http.StatusNoContent)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusNoContent)
 
 	pausedContainers, err := getSliceOfPausedContainers()
 
-	if err != nil {
-		c.Fatalf("error thrown while checking if containers were paused: %v", err)
-	}
+	c.Assert(err, checker.IsNil,Commentf("error thrown while checking if containers were paused: %v", err))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: `c.Assert(err, checker.IsNil, check.Commentf("error thrown while checking if containers were paused"))`

File: integration-cli/docker_api_containers_test.go [hunk 10]
```diff
@@ -483,39 +449,33 @@ func (s *DockerSuite) TestContainerApiPause(c *check.C) {
...
-	c.Assert(status, check.Equals, http.StatusNoContent)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusNoContent)
 
 	pausedContainers, err = getSliceOfPausedContainers()
 
-	if err != nil {
-		c.Fatalf("error thrown while checking if containers were paused: %v", err)
-	}
+	c.Assert(err, checker.IsNil,Commentf("error thrown while checking if containers were paused: %v", err))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Same as above (for err)

File: integration-cli/docker_api_containers_test.go [hunk 11]
```diff
@@ -523,29 +483,20 @@ func (s *DockerSuite) TestContainerApiTop(c *check.C) {
...
 
-	if err := json.Unmarshal(b, &top); err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(json.Unmarshal(b, &top),checker.IsNil)
 
-	if len(top.Titles) != 11 {
-		c.Fatalf("expected 11 titles, found %d: %v", len(top.Titles), top.Titles)
-	}
+	c.Assert(len(top.Titles),checker.Equals,11,Commentf("expected 11 titles, found %d: %v", len(top.Titles), top.Titles))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Use `checker.HasLen`

File: integration-cli/docker_api_containers_test.go [hunk 12]
```diff
@@ -523,29 +483,20 @@ func (s *DockerSuite) TestContainerApiTop(c *check.C) {
...
-	if len(top.Processes) != 2 {
-		c.Fatalf("expected 2 processes, found %d: %v", len(top.Processes), top.Processes)
-	}
-	if top.Processes[0][10] != "/bin/sh -c top" {
-		c.Fatalf("expected `/bin/sh -c top`, found: %s", top.Processes[0][10])
-	}
-	if top.Processes[1][10] != "top" {
-		c.Fatalf("expected `top`, found: %s", top.Processes[1][10])
-	}
+	c.Assert(len(top.Processes),checker.Equals,2,Commentf("expected 2 processes, found %d: %v", len(top.Processes), top.Processes))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Use `checker.HasLen`

File: integration-cli/docker_api_containers_test.go [hunk 13]
```diff
@@ -523,29 +483,20 @@ func (s *DockerSuite) TestContainerApiTop(c *check.C) {
...
+	c.Assert(top.Processes[0][10],checker.Equals,"/bin/sh -c top",Commentf("expected `/bin/sh -c top`, found: %s", top.Processes[0][10]))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: No need for comment here

File: integration-cli/docker_api_containers_test.go [hunk 14]
```diff
@@ -523,29 +483,20 @@ func (s *DockerSuite) TestContainerApiTop(c *check.C) {
...
+	c.Assert(top.Processes[1][10],checker.Equals,"top",Commentf("expected `top`, found: %s", top.Processes[1][10]))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Same here, no need for comment

File: integration-cli/docker_api_containers_test.go [hunk 15]
```diff
@@ -658,34 +593,29 @@ func (s *DockerSuite) TestContainerApiCreate(c *check.C) {
...
-		c.Fatal(err)
-	}
+
+	c.Assert(json.Unmarshal(b, &container),checker.IsNil)
 
 	out, _ := dockerCmd(c, "start", "-a", container.ID)
-	if strings.TrimSpace(out) != "/test" {
-		c.Fatalf("expected output `/test`, got %q", out)
-	}
+	c.Assert(strings.TrimSpace(out),checker.Equals,"/test",Commentf("expected output `/test`, got %q", out))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: No need for comment here.

File: integration-cli/docker_api_containers_test.go [hunk 16]
```diff
@@ -658,34 +593,29 @@ func (s *DockerSuite) TestContainerApiCreate(c *check.C) {
...
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusInternalServerError)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusInternalServerError)
 
 	expected := "Config cannot be empty in order to create a container\n"
-	if body := string(b); body != expected {
-		c.Fatalf("Expected to get %q, got %q", expected, body)
-	}
+	c.Assert(string(b),checker.Equals,expected,Commentf("Expected to get %q, got %q", expected,string(b)))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: No need for comment here

File: integration-cli/docker_api_containers_test.go [hunk 17]
```diff
@@ -697,26 +627,20 @@ func (s *DockerSuite) TestContainerApiCreateWithHostName(c *check.C) {
...
 	var containerJSON types.ContainerJSON
-	if err := json.Unmarshal(body, &containerJSON); err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(json.Unmarshal(body, &container),checker.IsNil)
 
-	if containerJSON.Config.Hostname != hostName {
-		c.Fatalf("Mismatched Hostname, Expected %s, Actual: %s ", hostName, containerJSON.Config.Hostname)
-	}
+	c.Assert(containerJSON.Config.Hostname,checker.Equals,hostName,Commentf("Mismatched Hostname, Expected %s, Actual: %s ", hostName, containerJSON.Config.Hostname))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]:
```go
c.Assert(containerJSON.Config.Hostname,checker.Equals,hostName,Commentf("Mismatched Hostname"))
```

File: integration-cli/docker_api_containers_test.go [hunk 18]
```diff
@@ -728,26 +652,20 @@ func (s *DockerSuite) TestContainerApiCreateWithDomainName(c *check.C) {
...
 	var containerJSON types.ContainerJSON
-	if err := json.Unmarshal(body, &containerJSON); err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(json.Unmarshal(body, &container),checker.IsNil)
 
-	if containerJSON.Config.Domainname != domainName {
-		c.Fatalf("Mismatched Domainname, Expected %s, Actual: %s ", domainName, containerJSON.Config.Domainname)
-	}
+	c.Assert(containerJSON.Config.Domainname,checker.Equals,domainName,Commentf("Mismatched Domainname, Expected %s, Actual: %s ", domainName, containerJSON.Config.Domainname))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]:
```go
c.Assert(containerJSON.Config.Domainname,checker.Equals,domainName,Commentf("Mismatched Domainname"))
```

File: integration-cli/docker_api_containers_test.go [hunk 19]
```diff
@@ -764,26 +682,20 @@ func UtilCreateNetworkMode(c *check.C, networkMode string) {
...
 	var containerJSON types.ContainerJSON
-	if err := json.Unmarshal(body, &containerJSON); err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(json.Unmarshal(body, &container),checker.IsNil)
 
-	if containerJSON.HostConfig.NetworkMode != runconfig.NetworkMode(networkMode) {
-		c.Fatalf("Mismatched NetworkMode, Expected %s, Actual: %s ", networkMode, containerJSON.HostConfig.NetworkMode)
-	}
+	c.Assert(containerJSON.HostConfig.NetworkMode,checker.Equals,runconfig.NetworkMode(networkMode),Commentf("Mismatched NetworkMode, Expected %s, Actual: %s ", networkMode, containerJSON.HostConfig.NetworkMode))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]:
```go
c.Assert(containerJSON.HostConfig.NetworkMode,checker.Equals,runconfig.NetworkMode(networkMode),Commentf("Mismatched NetworkMode"))
```

File: integration-cli/docker_api_containers_test.go [hunk 20]
```diff
@@ -868,14 +776,12 @@ func (s *DockerSuite) TestContainerApiInvalidPortSyntax(c *check.C) {
...
+	c.Assert(err, checker.IsNil)
+	c.Assert(res.StatusCode, checker.Equals, http.StatusInternalServerError)
 
 	b, err := readBody(body)
-	if err != nil {
-		c.Fatal(err)
-	}
-	c.Assert(strings.Contains(string(b[:]), "Invalid port"), check.Equals, true)
+	c.Assert(err, checker.IsNil)
+	c.Assert(strings.Contains(string(b[:]), "Invalid port"), checker.Equals, true)
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Use `checker.Contains` here

File: integration-cli/docker_api_containers_test.go [hunk 21]
```diff
@@ -906,39 +812,28 @@ func (s *DockerSuite) TestContainerApiPostCreateNull(c *check.C) {
...
 
 	out, err := inspectField(container.ID, "HostConfig.CpusetCpus")
-	if err != nil {
-		c.Fatal(err, out)
-	}
-	if out != "" {
-		c.Fatalf("expected empty string, got %q", out)
-	}
+	c.Assert(err, checker.IsNil)
+	c.Assert(out, checker.Equals,"",Commentf("expected empty string, got %q", out))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: No need for comment here

File: integration-cli/docker_api_containers_test.go [hunk 22]
```diff
@@ -952,14 +847,12 @@ func (s *DockerSuite) TestCreateWithTooLowMemoryLimit(c *check.C) {
...
 	b, err2 := readBody(body)
-	if err2 != nil {
-		c.Fatal(err2)
-	}
+	c.Assert(err2, checker.IsNil)
 
-	c.Assert(res.StatusCode, check.Equals, http.StatusInternalServerError)
-	c.Assert(strings.Contains(string(b), "Minimum memory limit allowed is 4MB"), check.Equals, true)
+	c.Assert(res.StatusCode, checker.Equals, http.StatusInternalServerError)
+	c.Assert(strings.Contains(string(b), "Minimum memory limit allowed is 4MB"), checker.Equals, true)
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Use `checker.Contains` here

File: integration-cli/docker_api_containers_test.go [hunk 23]
```diff
@@ -974,14 +867,12 @@ func (s *DockerSuite) TestStartWithTooLowMemoryLimit(c *check.C) {
...
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Use `checker.Contains` here

File: integration-cli/docker_api_containers_test.go [hunk 24]
```diff
@@ -991,14 +882,12 @@ func (s *DockerSuite) TestContainerApiRename(c *check.C) {
...
+	c.Assert(err, checker.IsNil)
 	// 204 No Content is expected, not 200
-	c.Assert(statusCode, check.Equals, http.StatusNoContent)
+	c.Assert(statusCode, checker.Equals, http.StatusNoContent)
 
 	name, err := inspectField(containerID, "Name")
-	if name != "/"+newName {
-		c.Fatalf("Failed to rename container, expected %v, got %v. Container rename API failed", newName, name)
-	}
+	c.Assert(name,checker.Equals,"/"+newName,Commentf("Failed to rename container, expected %v, got %v. Container rename API failed", newName, name))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]:
```go
c.Assert(name,checker.Equals,"/"+newName,Commentf("Failed to rename container"))
```

File: integration-cli/docker_api_containers_test.go [hunk 25]
```diff
@@ -1098,21 +980,15 @@ func (s *DockerSuite) TestContainerApiWait(c *check.C) {
...
-	if err := waitInspect(name, "{{ .State.Running  }}", "false", 5*time.Second); err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(waitInspect(name, "{{ .State.Running  }}", "false", 5*time.Second), checker.IsNil)
 
 	var waitres types.ContainerWaitResponse
-	if err := json.Unmarshal(body, &waitres); err != nil {
-		c.Fatalf("unable to unmarshal response body: %v", err)
-	}
+	c.Assert(json.Unmarshal(body, &waitres), checker.IsNil,Commentf("unable to unmarshal response body: %v",json.Unmarshal(body, &waitres)))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Same as above, about calling twice `json.UnMarshal`

File: integration-cli/docker_api_containers_test.go [hunk 26]
```diff
@@ -1098,21 +980,15 @@ func (s *DockerSuite) TestContainerApiWait(c *check.C) {
...
 
-	if waitres.StatusCode != 0 {
-		c.Fatalf("Expected wait response StatusCode to be 0, got %d", waitres.StatusCode)
-	}
+	c.Assert(waitres.StatusCode,checker.Equals,0,Commentf("Expected wait response StatusCode to be 0, got %d", waitres.StatusCode))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Comment is probably not needed here

File: integration-cli/docker_api_containers_test.go [hunk 27]
```diff
@@ -1308,31 +1175,21 @@ func (s *DockerSuite) TestContainersApiChunkedEncoding(c *check.C) {
 	req.ContentLength = -1
 
 	resp, err := client.Do(req)
-	if err != nil {
-		c.Fatalf("error starting container with chunked encoding: %v", err)
-	}
+	c.Assert(err, checker.IsNil,Commentf("error starting container with chunked encoding: %v", err))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: `c.Assert(err, checker.IsNil,Commentf("error starting container with chunked encoding"))`

File: integration-cli/docker_api_containers_test.go [hunk 28]
```diff
@@ -1308,31 +1175,21 @@ func (s *DockerSuite) TestContainersApiChunkedEncoding(c *check.C) {
...
 	resp.Body.Close()
-	if resp.StatusCode != 204 {
-		c.Fatalf("expected status code 204, got %d", resp.StatusCode)
-	}
+	c.Assert(resp.StatusCode,checker.Equals,204,Commentf("expected status code 204, got %d", resp.StatusCode))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: Comment is not needed.

File: integration-cli/docker_api_containers_test.go [hunk 29]
```diff
@@ -1308,31 +1175,21 @@ func (s *DockerSuite) TestContainersApiChunkedEncoding(c *check.C) {
...
-	if err := json.NewDecoder(strings.NewReader(out)).Decode(&binds); err != nil {
-		c.Fatal(err)
-	}
-	if len(binds) != 1 {
-		c.Fatalf("got unexpected binds: %v", binds)
-	}
+
+	c.Assert(json.NewDecoder(strings.NewReader(out)).Decode(&binds), checker.IsNil)
+
+	c.Assert(binds,checker.hasLen,1,"got unexpected binds: %v", binds)
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: s/hasLen/HasLen/

File: integration-cli/docker_api_containers_test.go [hunk 30]
```diff
@@ -1308,31 +1175,21 @@ func (s *DockerSuite) TestContainersApiChunkedEncoding(c *check.C) {
...
 
 	expected := "/tmp:/foo"
-	if binds[0] != expected {
-		c.Fatalf("got incorrect bind spec, wanted %s, got: %s", expected, binds[0])
-	}
+	c.Assert(binds[0],checker.Equals,expected,Commentf("got incorrect bind spec, wanted %s, got: %s", expected, binds[0]))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: `c.Assert(binds[0],checker.Equals,expected,Commentf("got incorrect bind spec"))`

File: integration-cli/docker_api_containers_test.go [hunk 31]
```diff
@@ -1521,11 +1376,9 @@ func (s *DockerSuite) TestContainersApiGetContainersJSONEmpty(c *check.C) {
...
 
 	status, body, err := sockRequest("GET", "/containers/json?all=1", nil)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusOK)
-	if string(body) != "[]\n" {
-		c.Fatalf("Expected empty response to be `[]`, got %q", string(body))
-	}
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusOK)
+	c.Assert(string(body),checker.Equals,"[]\n","Expected empty response to be `[]`, got %q", string(body))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: No need for comment here

File: integration-cli/docker_api_containers_test.go [hunk 32]
```diff
@@ -1537,10 +1390,10 @@ func (s *DockerSuite) TestPostContainersCreateWithWrongCpusetValues(c *check.C)
 	}{"busybox", "1-42,,"}
 	name := "wrong-cpuset-cpus"
 	status, body, err := sockRequest("POST", "/containers/create?name="+name, c1)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusInternalServerError)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusInternalServerError)
 	expected := "Invalid value 1-42,, for cpuset cpus.\n"
-	c.Assert(string(body), check.Equals, expected, check.Commentf("Expected output to contain %q, got %q", expected, string(body)))
+	c.Assert(string(body), checker.Equals, expected, check.Commentf("Expected output to contain %q, got %q", expected, string(body)))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: No need for comment here

File: integration-cli/docker_api_containers_test.go [hunk 33]
```diff
@@ -1548,8 +1401,8 @@ func (s *DockerSuite) TestPostContainersCreateWithWrongCpusetValues(c *check.C)
 	}{"busybox", "42-3,1--"}
 	name = "wrong-cpuset-mems"
 	status, body, err = sockRequest("POST", "/containers/create?name="+name, c2)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusInternalServerError)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusInternalServerError)
 	expected = "Invalid value 42-3,1-- for cpuset mems.\n"
-	c.Assert(string(body), check.Equals, expected, check.Commentf("Expected output to contain %q, got %q", expected, string(body)))
+	c.Assert(string(body), checker.Equals, expected, check.Commentf("Expected output to contain %q, got %q", expected, string(body)))
```
Comment (vdemeester) [inline integration-cli/docker_api_containers_test.go]: No need for comment here

**Comment by cpuguy83** [comment]
integration-cli/docker_api_containers_test.go: integration-cli/docker_api_containers_test.go:1158:68: expected ';', found '{'

**Comment by cpuguy83** [comment]
Please put spaces after commas in your arguments list.
I'm suprised go-fmt didn't pick this up.

**Comment by vdemeester** [comment]
@cpuguy83 I feel it didn't because of the previous message :

```bash
---> Making bundle: validate-gofmt (in bundles/1.9.0-dev/validate-gofmt)
<standard input>:1158:68: expected ';', found '{'
Congratulations!  All Go source files are properly formatted.
```

**Comment by calavera** [comment]
I'm very sorry but I'm going to close this PR until the errors are resolved. Please, feel free to send it again after testing that every test still works and the code is valid. `make test` and `make validate` are your friends.

As a rule of thumb, before opening all your PRs again, it will be easier to get them merged if you focus in one at a time, that way reviewers don't need to point the same errors over several changes.

**Pull request closed without merging**